examples/with_plugs.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Example OpenHTF test logic.
+
+Run with (your virtualenv must be activated first):
+
+  python with_plugs.py
+
+Afterwards, take a look at the hello_world.json output file.  This will
+give you a basic idea of what a minimal test outputs.
+
+For more information on measurements, see the measurements.py example.
+
+TODO(someone): Write an output example.
+For more information on output, see the output.py example.
+"""
+import subprocess
+import time
+
+import openhtf as htf
+from openhtf import plugs
+
+class PingPlug(plugs.BasePlug):
+  enable_remote = True
+  host = None
+  def __init__(self):
+    assert self.host is not None
+
+  def get_command(self, count):
+    return [
+      'ping',
+      '-c',
+      str(count),
+      self.host,
+    ]
+
+  def run(self, count):
+    command = self.get_command(count)
+    print "running: %s" % command
+    return subprocess.call(command)
+
+class PingGoogle(PingPlug):
```

Comment:
docstrings can be omitted on these subclasses, but can you add a comment at the top of this list to indicate what these are and why you need them?